web_src/js/features/projects.js

```diff
@@ -0,0 +1,97 @@
+const {csrf} = window.config;
+
+export default async function initProject() {
+  if (!window.config || !window.config.PageIsProjects) {
+    return;
+  }
+
+  const {Sortable} = await import(/* webpackChunkName: "sortable" */'sortablejs');
+  const boardColumns = document.getElementsByClassName('board-column');
+
+  for (const column of boardColumns) {
```

Comment:
```suggestion
  for (const column of boardColumns || []) {
```
safeguard

Reply:
what exactly does this do?

Reply:
Actually it's unnessary for `getElementsByClassName` as that will be always iterable, please resolve.

Reply:
@adelowo can resolve :)

Reply:
@6543 I am not certain what to do here. Seems @silverwind says it can say as.

> pretty late here so I might not be processing the conversation correctly :))